drop-cap/library/src/main/java/com/novoda/dropcap/DropCapView.java

```diff
@@ -6,15 +6,20 @@
 import android.graphics.Rect;
 import android.graphics.Typeface;
 import android.support.annotation.ColorInt;
-import android.support.annotation.DimenRes;
 import android.text.Layout;
 import android.text.StaticLayout;
 import android.text.TextPaint;
 import android.util.AttributeSet;
+import android.util.TypedValue;
 import android.view.View;
 
 public class DropCapView extends View {
 
+    private static final int TEXT_SET_INDEX = 0;
```

Comment:
This is actually not needed. You can add `<attr name="android:text" />` in the `DropCapView` stylable, and reference it with `R.styleable.DropCapView_android_text`. You save a lot of code 👅